sdv/model_generator/utils.py

```diff
@@ -12,10 +12,18 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-"""CodeGeneratorContext."""
+import re
 from typing import Optional
 
 
+def to_snake_case(camel_case_input):
```

Comment:
I expect that this only works for converting camel case to snake case. Maybe `camel_to_snake_case(input: str) -> str` would be a better way of calling this? Also please add type annotations